auditlog_tests/tests.py

```diff
@@ -634,7 +634,7 @@ def tearDown(self):
         super().tearDown()
 
     def test_model_with_same_time(self):
-        timestamp = datetime.datetime(2017, 1, 10, 12, 0, tzinfo=timezone.utc)
+        timestamp = datetime.datetime(2017, 1, 10, 12, 0, tzinfo=datetime.timezone.utc)
```

Comment:
We do `from datetime import timezone` everywhere else in the code, but in the tests we do just `import datetime`. Should we align the style to the former?

Reply:
Yes, I've changed it to `from datetime import timezone`